draft/0005-impersonation.md

```diff
@@ -242,14 +239,16 @@ Content-Type: application/json
 The resource owner API validates the support engineer's request and call Logto's Management API using a machine-to-machine application to issue a subject token for the user.
 
 ```
-POST /api/security/subject-tokens HTTP/1.1
-Host: example.logto.app
+POST /api/subject-tokens HTTP/1.1
+Host: example-tenant.logto.app
```

Comment:
Should we use some special mark to indicate that the value of `tenant` could vary? E.g. `example.{tenant}.logto.app`.

Reply:
I don't think so, this is just an example, it does not follow the tenant domain format actually.